index.bs

```diff
@@ -1270,7 +1270,7 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
         :   <dfn>userHandle</dfn>
         ::  The [=user handle=] associated when this [=public key credential source=] was created. This [=struct/item=] is
-            nullable.
+            nullable, however [=user handle=] MUST always be populated for [=discoverable credentials=].
```

Comment:
@Kieun made an interesting point about requiring that `userHandle` be a non-zero-length value instead of simply "not null" which would allow for `""` (an empty string):

https://github.com/w3c/webauthn/issues/1892#issuecomment-1552750057

Would this additional constraint be sufficient to prevent empty strings here?

Reply:
The userHandle cannot be the empty string - this is already transitively implied because:

1. That the userHandle is [by definition](https://w3c.github.io/webauthn/#user-handle) the value of user.id specified by the RP during credential creation, and;  
2. [the definiton of user.id](https://w3c.github.io/webauthn/#dom-publickeycredentialuserentity-id) says it `MUST NOT be empty`

As such, I don't see any real need to redundantly state this.